### DISCUSSION TITLE
Style elements without specifying a class name

### DISCUSSION THREAD

**Discussion by Gytjarek** [chakra-ui/panda#2780] [Q&A]
Is it possible to style all buttons or inputs in my project without specifying a class name?
```
<button>Button should be styled by using panda</button>
```
I know I can use global styles, but is there a better way? Can I create a button recipe and somehow inject it into global styling?

**Comment by segunadebayo** [comment]
I can't think of another way besides global styles.

**Comment by Gytjarek** [comment]
@segunadebayo and is that possible to extend/inject a recipe into a global styles somehow?
